src/SharedPipelineComponents/IlrSubmissionEvents/SFA.DAS.Provider.Events.Submission/SubmissionEventsProcessor.cs

```diff
@@ -109,16 +109,6 @@ private SubmissionEvent CalculateDelta(IlrDetails currentIlr, IlrDetails lastSee
                 (@event = @event ?? new SubmissionEvent()).PathwayCode = currentIlr.PathwayCode;
             }
 
-            if (currentIlr.ActualStartDate != lastSeenIlr?.ActualStartDate)
```

Comment:
Why was this removed?

Reply:
I believe it is because they are being handled at line 139-140 instead,
                || currentIlr.ActualStartDate != lastSeenIlr?.ActualStartDate
                || currentIlr.PlannedEndDate != lastSeenIlr?.PlannedEndDate
but if this is likely to be an issue elsewhere the original lines 112 - 121 can remain.

Reply:
As there is no impact, this will be reverted